packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -139,21 +163,12 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   );
 
   /* @conditional-compile-remove(video-background-effects) */
-  const [showVideoEffectsPane, setVideoEffectsPane] = useState(false);
-
-  /* @conditional-compile-remove(video-background-effects) */
-  const setShowVideoEffectsPane = useCallback(
-    (showVideoEffectsOptions: boolean): void => {
-      setVideoEffectsPane(showVideoEffectsOptions);
-    },
-    [setVideoEffectsPane]
-  );
+  const { toggleVideoEffectsPane } = useVideoEffectsPane(props.mobileView);
 
   const [showDrawer, setShowDrawer] = useState(false);
   const onMoreButtonClicked = useCallback(() => {
-    closePane();
```

Comment:
Chatted with @dmceachernmsft , this behavior (because of a regression in main branch) does not happen today so I will not fix in this PR, but will do a subsequent PR to fix this.